docs/ambiguity.md

```diff
@@ -7,9 +7,12 @@ Due to DST time changes, there is a possibility that a wall-clock time either do
 
 There are two mostly equivalent methods that accomplish this conversion: [`Temporal.DateTime.prototype.inTimeZone`](./datetime.html#inTimeZone) and [`Temporal.TimeZone.prototype.getAbsoluteFor`](./timezone.html#getAbsoluteFor).
 The `disambiguation` option to these methods controls what absolute time to return in the case of ambiguity:
-- `earlier` (the default): The earlier of two possible absolute times will be returned.
-- `later`: The later of two possible absolute times will be returned.
-- `reject`: A `RangeError` will be thrown.
+
+- `'compatible'` (the default): Acts like `'earlier'` for backward transitions like the start of DST in the Spring, and `'later'` for forward transitions like the end of DST in the Fall.
```

Comment:
I would just mention "acts like `'earlier'` for backward transitions and `'later'` for forward transitions", since transitions may also occur for reasons other than DST.
(also in several other places below)

Reply:
done